src/main/java/de/picturesafe/search/elasticsearch/impl/mapping/MappingResolver.java

```diff
@@ -0,0 +1,166 @@
+        final String type = stringValue(properties, "type");
+        final boolean enabled = booleanValue(properties, "enabled", true);
+        final FieldConfiguration fieldConfiguration;
+
+        if (type == null) {
+            if (!enabled) {
+                fieldConfiguration = StandardFieldConfiguration.builder(name, ElasticsearchType.OBJECT).withoutIndexing().build();
+            } else {
+                return resolveObjectField(name, properties);
+            }
+        } else if (type.equals("nested")) {
+            return resolveNestedField(name, properties);
+        } else if (type.equals("completion")) {
+            fieldConfiguration = SuggestFieldConfiguration.name(name);
+        } else {
+            fieldConfiguration = fieldConfiguration(name, type, properties, enabled);
+        }
+        return new MappingField(fieldConfiguration, Collections.emptyList());
+    }
+
+    private static FieldConfiguration fieldConfiguration(String name, String type, Map<String, Object> properties, boolean enabled) {
+        final boolean sortableAndAggregatable = isSortableAndAggregatable(properties);
+        return StandardFieldConfiguration.builder(name, type)
+                    .sortable(sortableAndAggregatable)
+                    .aggregatable(sortableAndAggregatable)
+                    .analyzer(stringValue(properties, "analyzer"))
+                    .copyTo((Collection<String>) properties.get("copy_to"))
+                    .additionalParameters(additionalParameters(properties))
+                    .withoutIndexing(!enabled)
+                    .build();
+    }
+
+    private static MappingField resolveObjectField(String name, Map<String, Object> doc) {
+        final List<Locale> locales = new ArrayList<>();
+        final MutableObject<FieldConfiguration> mutableFieldConfig = new MutableObject<>();
+        objectValue(doc, "properties").forEach((subName, properties) -> {
+            final Locale locale = locale(subName);
+            if (locale != null) {
+                locales.add(locale);
+            }
+            final MappingField field = resolveField(subName, (Map<String, Object>) properties);
+            if (mutableFieldConfig.getValue() == null) {
+                mutableFieldConfig.setValue(field.getFieldConfiguration());
+            } else if (!field.getFieldConfiguration().equalsBesidesName(mutableFieldConfig.getValue())) {
+                throw new RuntimeException("Object fields are only supported as multilingual text fields or with enabled=false!");
+            }
+        });
+        final FieldConfiguration fieldConfiguration = StandardFieldConfiguration.builder(name, mutableFieldConfig.getValue()).multilingual(true).build();
+        return new MappingField(fieldConfiguration, locales);
+    }
+
+    private static Locale locale(String name) {
+        try {
+            return LocaleUtils.toLocale(name);
+        } catch (Exception e) {
+            return null;
```

Comment:
Maybe logging exception on debug level?

Reply:
I do not see a reason why. This is expected behavior, if the object field is not part of a multilingual field, there will be no locale.

Reply:
ok!